src/map_homework/util/ArrayListUtils.java

```diff
@@ -0,0 +1,55 @@
+package map_homework.util;
+
+import map_homework.model.Faculties;
+import map_homework.model.Student;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ArrayListUtils {
+
+    public static HashMap getCountOfStudentsOccurrences(ArrayList<Student> studentsList) {
+        HashMap<Student, Integer> tempMap = new HashMap<>();
+        for (Student student : studentsList) {
+            if (tempMap.containsKey(student)) {
+                tempMap.put(student, tempMap.get(student) + 1);
+            } else {
+                tempMap.put(student, 1);
+            }
+        }
+        return tempMap;
+    }
+
+    public static HashMap getCountOfStudentsInFaculties(ArrayList<Student> studentsList) {
+        HashMap<Faculties, Integer> tempMap = new HashMap<>();
+        for (Student student : studentsList) {
+            Integer countOfStudentsInFaculties = tempMap.get(student.getFaculty());
+            if (countOfStudentsInFaculties == null) {
+                tempMap.put(student.getFaculty(), 1);
+            } else {
+                tempMap.put(student.getFaculty(), countOfStudentsInFaculties + 1);
+            }
+        }
+        return tempMap;
+    }
+
+    public static void printSameStudentsOccurrencesMap(HashMap<Student, Integer> map) {
+        for (Map.Entry<Student, Integer> entry : map.entrySet()) {
+            System.out.println(entry.getKey() + " " + entry.getValue());
+        }
+        System.out.println();
+    }
+
+    public static void printFacultyMembersQuantityMap(HashMap<Faculties, Integer> map) {
```

Comment:
In case there is no student in a Faculty will this method print number of students in that Faculty?

Reply:
I didn't mean no student at all, I mean like in case there are 10 students in LAW faculty, 10 students in PHYSICS and no student in MATH, will this method print MATH? e.g.
`LAW 10`
`PHYSICS 10`
`MATH 0`